src/Contract/Config/ConfigurableCollectorConfig.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Contract\Config;
+
+abstract class ConfigurableCollectorConfig extends CollectorConfig
+{
+    private const ESCAPEES = [
```

Comment:
Can we have a key->value array instead of 2 lists? It would make it IMHO more readable and less prone to error.

Reply:
This data is borrowed from symfony and I don't really know what all codes do. So wouldn't change it.

Reply:
You are just replacing the found instances of `ESCAPEES` with `ESCAPED` at the same position in the array. If it was just one key->value array you would just change it to `str_replace(array_keys(self::ARRAY), array_values(self::ARRAY), $regex)`. That would be more readable to me. Not a blocker though.